src/modal/styles.scss

```diff
@@ -106,6 +106,12 @@ $modal-z-index: 5000;
 .footer {
   @include container.divider;
   padding: container.$footer-padding;
+  background-color: awsui.$color-background-container-content;
+  border-radius: 0 0 awsui.$border-radius-container awsui.$border-radius-container;
```

Comment:
When the Modal is used with `disableContentPaddings`, this border radius lets the content shine through. Can we either apply the border radius only when the Modal has been scrolled all the way, or remove the border radius completely and add `overflow:hidden` to the Modal?

![Screenshot of a testing page of the Modal component, showing a bit of text in the corner of the footer.](https://user-images.githubusercontent.com/9086585/227550200-1e34f925-f0d8-450a-9d50-4aa016950c44.jpg)

Reply:
good catch! I will look into that.